.github/workflows/build.yml

```diff
@@ -129,11 +129,11 @@ jobs:
         uses: microsoft/setup-msbuild@v1.0.2
         if: runner.os == 'Windows'
       - name: Install deps
-        run: python -m pip install -U cibuildwheel==1.9.0
+        run: python -m pip install -U cibuildwheel==2.1.2
       - name: Build Wheels
         env:
           CIBW_BEFORE_ALL_LINUX: "yum install -y openblas-devel"
-          CIBW_SKIP: "cp27-* cp34-* cp35-* pp*"
```

Comment:
How come we don't need to skip these older versions anymore? Are they automatically skipped?

Reply:
Yeah cibuildwheel dropped support for building wheels for python < 3.6 in 2.0.0: https://github.com/pypa/cibuildwheel/releases/tag/v2.0.0